packages/framework/src/Component/Domain/Domain.php

```diff
@@ -154,6 +154,18 @@ public function getDomainConfigById($domainId)
         throw new \Shopsys\FrameworkBundle\Component\Domain\Exception\InvalidDomainIdException();
     }
 
+    /**
+     * @return int[][]
```

Comment:
```suggestion
     * @return int[]
```

We are trying to avoid this

Reply:
Ok I will chaange that to `array` as your suggestion is not valid.